wtforms_hcaptcha/widgets.py

```diff
@@ -3,7 +3,7 @@
 from markupsafe import Markup
 
 HCAPTCHA_HTML = Markup("""<div class="h-captcha" data-sitekey="{site_key}"></div>
-<script src="https://hcaptcha.com/1/api.js" async defer></script>
+<script defer src="https://hcaptcha.com/1/api.js" async defer></script>
```

Comment:
This script tag already has a defer attribute

Reply:
sry for mistake with defer) pagespeed show warning and i didnt see right side(